seaborn/distributions.py

```diff
@@ -1770,6 +1827,57 @@ def kdeplot(
 )
 
 
+def ecdfplot(
+    data=None, *,
+    # Vector variables
+    x=None, y=None, hue=None, weights=None,
+    # Computation parameters
+    stat="proportion",
+    # Hue mapping parameters
+    palette=None, hue_order=None, hue_norm=None, color=None,
+    # Axes information
+    log_scale=None, legend=True, ax=None,
+    # Other appearance keywords
+    **kwargs,
+):
```

Comment:
Suggested docstring:

```suggestion
):
    Plot ECDF of data.
    
    :param data: Tidy (“long-form”) dataframe where each column is a variable and each row is an observation.
    :param x: Column name of variable to plot ECDF for.
    :param hue: Grouping variable that will produce elements with different colors. Can be either categorical or numeric, although color mapping will behave differently in latter case.
    :param weights: Column that houses weighting for each sample.
    :param stat: Whether to plot proportion or raw counts. Defaults to "proportion".
```

Apologies, I have to drop off for a bit, so my review comment ends here.

Reply:
Thanks — yeah, filling in the docstrings etc. is on the TODO, just wanted to get the PR started. The seaborn docs use a kind of convoluted templating system for maintaining consistency across parameters in the shared API, so most of the doc are actually written, they just need to be stuck together.